interface/billing/sl_eob_process.php

```diff
@@ -621,6 +622,28 @@ function era_callback(&$out)
                     );
                 }
             }
+
+            if ($out['corrected'] == '1') {
+                if ($GLOBALS['update_mbi']) {
+                    if ($primary) {
+                        InsuranceService::update($pid, "primary", array($insurance_id, $out['corrected_mbi']));
+                    } else { // secondary medicare
+                        InsuranceService::update($pid, "secondary", array($insurance_id, $out['corrected_mbi']));
+                    }
+
```

Comment:
looks like this function needs every data element provided, would it be ok to add a function to the class that sets only one of the fields?

Reply:
going to keep debugging to amke sure $insurance_id is set